framework/rest/IndexAction.php

```diff
@@ -30,6 +30,28 @@ class IndexAction extends Action
      * The callable should return an instance of [[ActiveDataProvider]].
      */
     public $prepareDataProvider;
+    /**
+     * @var string class name of the search class related to this model. It can be optionally involved but not required.
+     * The called search class should implement [[\yii\data\SearchInterface]] to prepare an instance of [[ActiveDataProvider]].
+     * If not set, the default data provider returned by [[prepareDataProvider()]] will be used instead.
+     *
+     * Note that [[\yii\base\Model::formName()]] is used by default in a search class generated by **gii** when retrieving the attributes names from the URL query request.
+     * If the URL endpoint should look like `/resources?attribute=value` instead of `/resources?FormName[attribute]=value`
+     * then be sure that your search class is using `load($params, '')` instead of `load($params)`.
+     * @see \yii\base\Model::load()
```

Comment:
`@see` does not work in the middle of a description. Should be a sentence `See [[\yii\base\Model::load()]] for more details.` instead.